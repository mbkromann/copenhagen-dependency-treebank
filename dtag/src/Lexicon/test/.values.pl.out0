list([a,b,c]-[a], 1)
set([d,e,f]-[f], 2)
hash([g=1,h=2,i=3]-[g], 3)
list([a,b,c]-[a], 0)
list([a,b,c]-[], 0)
