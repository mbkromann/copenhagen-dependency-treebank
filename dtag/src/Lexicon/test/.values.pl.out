ERROR! cannot read file /opt/dta/arcs.header in Graph->readfile
check that DTAGHOME is set correctly!
ERROR! cannot read file /opt/dta/arcs.trailer in Graph->readfile
check that DTAGHOME is set correctly!
list([a,b,c]-[a], 1)
set([d,e,f]-[f], 2)
hash([g=1,h=2,i=3]-[g], 3)
list([a,b,c]-[a], 0)
list([a,b,c]-[], 0)
